app/templates/emails/clarification_question.html

```diff
@@ -6,7 +6,7 @@
 <body>
 Supplier ID: {{ supplier_id }}
 <br />
-User name: {{ user_name }}
+User ID: {{ user_name }}
```

Comment:
I think we just want to drop user altogether and only have Supplier ID.  I don't see any value in showing two IDs here?